hugo.toml

```diff
@@ -17,6 +17,9 @@ disableBrowserError = true
 
 disableKinds = ["taxonomy"]
 
+# For offline build we need to ignore getJson error and handle nil returned instead. All getJson usages should be covered with warns
+ignoreErrors = ["error-remote-getjson"]
```

Comment:
Ideally, we should also revise `layouts/shortcodes/release-binaries.html` and `layouts/_default/cve-feed.json`.

Reply:
Fixed